services/startup.sh

```diff
@@ -25,6 +25,9 @@ touch "${once}"
 # Use default directory if SERVICE_DIR is not provided.
 SERVICE_DIR="${SERVICE_DIR-/media/developer/apps/usr/palm/services/org.webosbrew.hbchannel.service}"
 
+# Add service bin directory to PATH for devices without telnetd.
+export PATH="${PATH}:${SERVICE_DIR}/bin"
```

Comment:
Should we just explicitly specify `${SERVICE_DIR}/bin/telnetd` in `startup.sh` and leave `PATH` alone? This would ensure everyone is using the same `telnetd` and also prevent issues if LG were to add back a broken binary.

Reply:
I would prefer your idea of just using shipped `telnetd`.